fly.io/cdn-2024-01-26/fly.toml

```diff
@@ -0,0 +1,43 @@
+# Full app config reference: https://fly.io/docs/reference/configuration/
+app = "cdn-2024-01-26"
+# Closest to James
+primary_region = "sjc"
+# Secondary region will be "lhr", closest to Gerhard
```

Comment:
Note to self: change this to `iad` so that it's in the same region as the origin.